src/course-outline/utils.tsx

```diff
@@ -38,13 +42,12 @@ const getItemStatus = ({
 
 /**
  * Get section badge status content
- * @param {string} status - value from on getItemStatus util
- * @returns {
- *   badgeTitle: string,
- *   badgeIcon: node,
- * }
  */
-const getItemStatusBadgeContent = (status, messages, intl) => {
+const getItemStatusBadgeContent = (
+  status: ItemBadgeStatusValue,
+  messages: Record<string, MessageDescriptor>,
+  intl: IntlShape,
+): any => {
```

Comment:
Could you check if we can leave the type implicitly here?
```suggestion
) => {
```